.travis.yml

```diff
@@ -6,6 +6,7 @@ script:
   - bundle exec rake db:migrate RAILS_ENV=test
   - bundle exec rspec -fd
 before_script:
+  - cp config/application.travis.yml config/application.yml
```

Comment:
I've stopped using Figaro and use dotenv instead... I think it works a little nicer for use with heroku.

Probably not an issue, just an FYI.